src/FourBitTwoDisclosureDeviceUnlocker.java

```diff
@@ -205,11 +202,7 @@ private static boolean isValidSpin(final int numOfSpins) {
     private static CharSequence doPeek(final CharSequence pattern) {
         CharSequence returnPattern;
         boolean validPattern = isPeekValid(pattern);
```

Comment:
I think this is my code, which I can change if you want. But we can remove validPattern and just used isPeekValid(pattern) in the if statement.

Reply:
Done! Easy to fix since it's already open :)